onchain/src/social/request.cairo

```diff
@@ -18,10 +28,15 @@ pub struct SocialRequest<C, +Display<C>> {
 
 pub fn verify<C, +Display<C>>(request: @SocialRequest<C>) -> bool {
     // TODO: implement verification
-    // println!("{}", request.pubkey);
-    // println!("{}", request.content);
-    // println!("{:?}", request.sig);
-    false
+    println!("{}", request.pubkey);
+    println!("{}", request.content);
+    println!("{:?}", request.sig);
+
+    let valid: bool = bip340::verify(
```

Comment:
The signature should be verified against the message hash as described in NIP-01:
https://github.com/nostr-protocol/nips/blob/master/01.md#events-and-signatures

Reply:
Alright